docs/troubleshooting.md

```diff
@@ -28,6 +28,17 @@ chmod -R g+w <path to directory>
 # Extend the execution and directory browsing permissions that the owner has in a directory to everyone.
 find <path to directory> -executable -exec chmod a+x {} \;
 ```
+### Are GPU drivers installed everywhere?
+
+Enrico writes that 
+
+```text
+the cuda version output from nvidia-smi doesn't mean necessarily that cuda is installed. It simply tells you which cuda release matches the installed drivers```
+
+on slurm nodes, cuda is not preinstalled because I encourage people to install it in their own conda environments so that they can keep it stable
+
+On non-slurm nodes, i.e. g1-7 and g1-9 currently, cuda is still preinstalled
+```
```

Comment:
Question and answer don't line up here: the question is about the drivers while the answer is about cuda. All users need to know is the following: drivers are preinstalled, while cuda isn't. Users should install cuda themselves.